src/DetectChanges/BCBreak/FunctionBased/ParameterNameChanged.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Roave\BackwardCompatibility\DetectChanges\BCBreak\FunctionBased;
+
+use Roave\BackwardCompatibility\Change;
+use Roave\BackwardCompatibility\Changes;
+use Roave\BackwardCompatibility\Formatter\ReflectionFunctionAbstractName;
+use Roave\BetterReflection\Reflection\ReflectionFunctionAbstract;
+use Roave\BetterReflection\Reflection\ReflectionParameter;
+
+use function array_intersect_key;
+use function Safe\sprintf;
+
+/**
+ * @todo only apply this to PHP 8+ code
```

Comment:
`@todo`

Reply:
I think the only way we can realistically do this is to:

 - Write a class like `ReadPhpVersionRequirementsFromComposerJson` that:
   - reads `composer.json`
   - find the `require.php` key and stores the min/max value
   - it would be run lazily
 - The class can be constructor-injected into the `ParameterNamedChanged` check
 - The `ParameterNamedChanged` can ask the class if we intersect PHP 8+ at all, if not, return

Reply:
IMO this also applies to PHP 7.x code - running the same code on 8.x will have these issues, regardless if we're developing on 7.x right now

Reply:
17adbea4322a7eddbc19f59e14828c5f69eac9a6